src/Encoding.ts

```diff
@@ -35,7 +35,7 @@ class Encoding {
     const mapped = this.unicodeMappings[codePoint];
     if (!mapped) {
       const str = String.fromCharCode(codePoint);
-      const msg = `${this.name} cannot encode "${str}"`;
+      const msg = `${this.name} cannot encode "${str}" (0x${codePoint.toString(16).padStart(2, '0')})`;
```

Comment:
This is a nice addition. I would like to avoid using `String.padStart` though. `pdf-lib` uses a custom `padStart` function to make sure it is available in all browsers. Perhaps this isn't necessary anymore, but regardless I'd like to ensure consistency between the codebases when possible. 
```js
export const padStart = (value: string, length: number, padChar: string) => {
  let padding = '';
  for (let idx = 0, len = length - value.length; idx < len; idx++) {
    padding += padChar;
  }
  return padding + value;
};
```
> https://github.com/Hopding/pdf-lib/blob/master/src/utils/strings.ts#L14-L20.

Reply:
Thanks for the quick response (and for the _awesome_ library!). 

I am not sure how to both avoid repeating the code above and also avoid a circular dependency on `pdf-lib`. NodeJS@14 now complains about circular dependency trees.

I could copy and paste the `padStart` function, although this is potentially not great for the library's maintainability and minimising code base size (albeit this is a fairly trivial function).

another easy option which is backward compatible is:
```javascript
('000' + codePoint.toString(16)).slice(-4)
```
Although of course this always displays a 4 digit hex code, which might not be in keeping with the rest of the library.